ui/raidboss/data/04-sb/dungeon/temple_of_the_fist.txt

```diff
@@ -0,0 +1,137 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / is no longer sealed/ window 5000 jump 0
+
+# Coeurl Twins
+# -ii 1FD2 1FD8
+
+0.0 "--sync--" sync /Tourmaline Pond will be sealed off/
+3.3 "--sync--" sync /:Coeurl Sruti:366:/ window 3,1
+6.4 "Pounce" sync /:Coeurl Sruti:1FD1:/
+17.8 "Radial Blaster" sync /:Coeurl Sruti:1FD3:/
+22.0 "Pounce" sync /:Coeurl Sruti:1FD1:/
+
+38.5 "--Smriti Appears--" sync /:Added new combatant Coeurl Smriti/ window 10,10
+46.5 "Wide Blaster" sync /:Coeurl Smriti:1FD4:/ window 15,15
+46.5 "Radial Blaster" sync /:Coeurl Sruti:1FD3:/
+52.6 "Pounce x2" sync /:Coeurl Smriti:1FD1:/
+60.7 "Pounce x2" sync /:Coeurl Smriti:1FD1:/
+
+72.3 "Wide Blaster" sync /:Coeurl Smriti:1FD4:/ window 15,15
+72.3 "Radial Blaster" sync /:Coeurl Sruti:1FD3:/
+78.4 "Pounce x2" sync /:Coeurl Smriti:1FD1:/
+86.5 "Pounce x2" sync /:Coeurl Smriti:1FD1:/ jump 60.7
+
+98.1 "Wide Blaster"
+98.1 "Radial Blaster"
+104.2 "Pounce"
+112.3 "Pounce"
+123.9 "Wide Blaster"
+123.9 "Radial Blaster"
+
+# Phase change after one dies
+150.0 "Basic Instinct" sync /:Coeurl (?:Sruti|Smriti):1FD5:/ window 150,10
```

Comment:
Did `test_timeline` complain about the `?:`?

Reply:
It did. I was just looking through that for another timeline. It's a *bit* of a long stack trace. Is it better to just use a capture there, or use an invisible second line, do you think?

Reply:
I think in the short term, better to use capture.  `test_timeline` could be updated to understand non-capturing groups in the future too.